services/hanke-service/src/test/resources/application-test.yml

```diff
@@ -13,6 +13,9 @@ haitaton:
   features:
     hanke-editing: true
     user-management: true
+  profiili-api:
```

Comment:
Where are these used?

Reply:
They are loaded into ProfiiliProperties.

Reply:
I meant the values `http://localhost:14678/*`. What service is this? And why these properties are under `test` when there is no other changes there? All other test changes are under `integrationTest`.

Reply:
I was planning on using that port for the mockWebService in ProfiiliClientITest, but I used two random ports instead. These could probably be replaced with tokens that are clearly non-functional. There should be something here to override the default value in application.yml, so that we don't accidentally connect to the actual backend from integration tests.

Both tests and integration tests use the test Spring profile, so they use the same application-test.yml. Integration tests can see this file from the src/test/ directory, but tests can't access anything in src/intergationTest/, so the file needs to be here under src/test/.